inputs/worker_inner:6.json

```diff
@@ -3,6 +3,9 @@
     {
       "name": "reactor_config"
     },
+    {
+      "name": "check_and_set_platforms"
+    },
```

Comment:
Will anything in the worker build need this? I think it should probably only run in the orchestrator build.

Reply:
Leaving alone for now.